File: app/src/main/java/com/wavecat/inline/preferences/Column.java
package com.wavecat.inline.preferences;

import android.annotation.SuppressLint;
import android.content.Context;

import org.luaj.vm2.LuaTable;

@SuppressLint("ViewConstructor")
public class Column extends LinearGroup {

    public Column(Context context, LuaTable views) {
        super(context, views);
        setOrientation(VERTICAL);
    }
}

File: app/src/main/java/com/wavecat/inline/preferences/views/Column.kt
package com.wavecat.inline.preferences.views

import android.annotation.SuppressLint
import android.content.Context
import org.luaj.vm2.LuaTable

/**
 * A [LinearGroup] that arranges its children vertically.
 *
 * @param context The context.
 * @param views A [LuaTable] containing the views to be added to this layout.
 */
@SuppressLint("ViewConstructor")
class Column(
    context: Context,
    views: LuaTable,
) : LinearGroup(context, views) {
    init {
        orientation = VERTICAL
    }
}
